mailpoet/assets/js/src/newsletters/models/newsletter.ts

```diff
@@ -1,4 +1,20 @@
-type NewsLetterType = 'standard';
+export enum NEWSLETTER_TYPE {
+  STANDARD = 'standard',
+  AUTOMATIC = 'automatic',
+  WELCOME = 'welcome',
+  NOTIFICATION = 'notification',
+  NOTIFICATION_HISTORY = 'notification_history',
+  WC_TRANSACTIONAL = 'wc_transactional',
+  RE_ENGAGEMENT = 're_engagement'
+}
```

Comment:
This is just a stylistic note but we should preserve the same style as for other enums. This means `enum NewsletterType` and enum fields `Standard`, `Automatic`, etc.

<img width="530" alt="Screen Shot 2022-03-11 at 13 41 32" src="https://user-images.githubusercontent.com/141436/157868645-40c755f2-52ef-44ea-80e0-38069fbbe3ba.png">
<img width="240" alt="Screen Shot 2022-03-11 at 13 42 28" src="https://user-images.githubusercontent.com/141436/157868782-18658034-0328-4f43-a96c-037a2f7adff9.png">